UnityProject/Assets/Scripts/Medical/CloningConsole.cs

```diff
@@ -58,18 +70,29 @@ public class CloningRecord
 	public string ScanID;
 	public float OxyDmg;
 	public float BurnDmg;
-	public float ToxingDmg;
+	public float ToxinDmg;
 	public float BruteDmg;
 	public string UniqueIdentifier;
+	public CharacterSettings characterSettings;
+	public int mobID;
+	public Mind mind;
 
-	public CloningRecord(string name, int scanID, float oxyDmg, float burnDmg, float toxingDmg, float bruteDmg, string uniqueIdentifier)
+	public CloningRecord()
 	{
-		Name = name;
+		int scanID = Random.Range(0, 1000);
```

Comment:
there's a really high chance of same id being generated, what happens if two records have the same id?
maybe just start from 0001 and keep incrementing it instead?